docs/products/kafka/concepts/acl.rst

```diff
@@ -0,0 +1,159 @@
+Access control lists permission mapping
+=======================================
+
+Aiven for Apache Kafka supports the definition of users and access control lists (ACL) to allow and limit producer and consumer rights at topic level. You can manage users and ACL entries in the corresponding tabs of the service page in the Aiven web console as explained in the :doc:`dedicated documentation <../howto/manage-acls>`.
```

Comment:
can we use the same "Aiven for Apache Kafka uses access control lists (ACL) and user definitions in order to establish individual rights to produce or consume a topic." as in other place?